src/components/dev-hub/card-list.js

```diff
@@ -38,12 +40,16 @@ export default ({ items = [], limit = 9 }) => {
             <CardContainer>
                 {items.slice(0, visibleCards).map(item => (
                     <ArticleCard
-                        key={item['meta-description']}
-                        image={item['atf-image']}
-                        tags={item.languages}
-                    >
-                        {item['meta-description']}
-                    </ArticleCard>
+                        key={item['_id']}
+                        image={withPrefix(item['atf-image'])}
+                        tags={[
```

Comment:
I'm adding some helpers for the tags in my PR [here](https://github.com/mongodb/devhub/pull/88/files#diff-6b2033282b007c4d3f0b56a23c5b8e02), since these fields could be optional!

Reply:
Yes, please add a comment here so we don't forget to update it. If there is an article without a language tag it'll crash the renderer

Reply:
updated with the new method! thanks